source/content.js

```diff
@@ -199,6 +200,15 @@ function ajaxedPagesHandler() {
 		enableFeature(openCIDetailsInNewTab);
 		enableFeature(waitForBuild);
 		enableFeature(toggleAllThingsWithAlt);
+
+		observeEl('.js-discussion', () => {
+			enableFeature(hideInactiveDeployments);
+		}, {
+			childList: true,
+			attributes: true,
+			subtree: true,
+			attributeFilter: ['class']
+		});
```

Comment:
This should be in the feature file. You should only have `enableFeature(hideInactiveDeployments);` here.